src/main/kotlin/world/trait/SysTrait.kt

```diff
@@ -21,4 +23,17 @@ class SysTrait : Trait("sys") {
         return super.getProp(c, name)
     }
 
+    override fun callFunc(c: Context, name: String, args: List<Value>): Value? {
+        when (name) {
+            "addTrait" -> return funcAddTrait(c, args)
+        }
+        return null
+    }
+
+    private fun funcAddTrait(c: Context, args: List<Value>): Value {
```

Comment:
Is there a reason you decided to prefix the name with `func`?  I'm a fan of brevity, but I'm fine with whatever naming convention you prefer.

Reply:
Yeah, I had a standard going on all the classes that dispatch hard-coded function calls, they're all named funcWhatever.  it's mostly to remind future-us that "these are called by the func dispatcher, not like, normal funcs you might call for some other reason".

Reply:
> Is there a reason you decided to prefix the name with `func`? I'm a fan of brevity, but I'm fine with whatever naming convention you prefer.

This actually brings up a larger issue -- 'func' is what I've been calling what a 'verb' would be in MOO server.  But there's really no good reason, maybe I should just give in and use that nomenclature.  I keep having to correct myself mentally anyway.